### PR TITLE
DOC url building

### PR DESCRIPTION
**Pull request by bealdav**

### CODE REVIEW

**Comment by hparfr** [comment]
I'm not even sure if we should remove this feature completely.

**Comment by bealdav** [comment]
personnally I like this feature, let's discuss about it in real life

File: docky/common/project.py [hunk 1]
```diff
@@ -44,11 +44,13 @@ def get_containers(self, service=None):
     def show_access_url(self):
```
Comment (hparfr) [inline docky/common/project.py]: rename to display_help_tooltip() or something ?

**Comment by bealdav** [comment]
please squash and merge, thanks

**Pull request merged**